assets/src/components/conversations/ConversationsProvider.tsx

```diff
@@ -374,19 +393,8 @@ export class ConversationsProvider extends React.Component<Props, State> {
       },
       {}
     );
-    const sortedConversationIds = Object.keys(conversationsById).sort(
-      (x: string, y: string) => {
-        const a = conversationsById[x];
-        const b = conversationsById[y];
-        const left = a.last_activity_at
-          ? +new Date(a.last_activity_at)
-          : -Infinity;
-        const right = b.last_activity_at
-          ? +new Date(b.last_activity_at)
-          : -Infinity;
-
-        return right - left;
-      }
+    const sortedConversationIds = this.getSortedConversationIds(
```

Comment:
Note that I migrated the logic for sorting conversations into `getSortedConversationIds`.